specifications/api/swagger.yaml

```diff
@@ -2381,6 +2394,16 @@ components:
       description: |
         Withdrawals that could be external or belong to the wallet.
 
+    ApiMintedBurntAssets: &ApiMintedBurntAssets
+      description: |
+        Assets minted (created) or burnt (destroyed)
+        This amount contributes to the total transaction value.
+        Positive values denote creation of assets and negative values
+        denote the reverse.
```

Comment:
Given that you have two fields of this type, I think it would be okay to only allow positive values here.

Reply:
revised this fragments